internal/provider/private_endpoint_services_resource.go

```diff
@@ -252,21 +253,23 @@ func loadEndpointServicesIntoTerraformState(
 			EndpointServiceId: types.StringValue(service.GetEndpointServiceId()),
 		}
 
-		if services[i].CloudProvider.String() == "AWS" {
```

Comment:
Is this the root cause of the original bug?  This clause was getting skipped because .String() is used instead of .ValueString()?

Reply:
Yes

Reply:
I think it was me that missed that during review.   The terraform api makes this an easy mistake to make.